serverless/functions/app/token.js

```diff
@@ -1,4 +1,7 @@
 exports.handler = function (context, event, callback) {
+  const verifyExpiry = require(Runtime.getAssets()['/verify_expiry.js'].path);
+  verifyExpiry.handler(context, event, callback);
+
   const AccessToken = Twilio.jwt.AccessToken;
```

Comment:
I just realized, if we keep this as is (which is ok to me), how do we make sure the rest of the handler code doesn't execute if `verifyExpiry.handler` returns a failure code? In the current implementation, lines 5 onwards will still execute even if `verifyExpiry.handler` executes the callback.

Reply:
Great question. I think we are ok. Per the [docs](https://www.twilio.com/docs/runtime/functions/invocation):

> When your Twilio Function code has completed, you call the callback() method to emit your response. After executing the callback() method, your Twilio Function execution will be terminated. This includes any asynchronous processes that may still executing.

Reply:
Ok cool.